website/docs/cli/commands/add.html.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: "docs"
+page_title: "Command: add"
+sidebar_current: "docs-commands-add"
+description: |-
+  The `terraform add` command generates resource configuration templates.
+---
+
+# Command: add
+
+The `terraform add` command generates a resource configuration template with
+`null` placeholder values for all attributes, unless the `-from-state` flag is
+used. By default, the template only includes required resource attributes; the
+`-optional` flag tells Terraform to also include any optional attributes. 
+
+## Usage
+
+Usage: `terraform add [options] ADDRESS`
+
+This command requires an address that points to a resource which does not
+already exist in the configuration. Addresses are in 
+[resource addressing format](/docs/cli/state/resource-addressing.html).
+
+The command-line flags are all optional. The list of available flags are:
```

Comment:
While revising these we've been gradually changing the typical wording of this paragraph to say "This command accepts the following options:", in order to get the same effect with fewer words. Not a huge deal, but would be nice to start this one off in the new shape. :smile: 

The doc page for `terraform taint` is an example of one we recently refreshed as part of the `terraform plan -replace=...` work, in case you'd like to see a full example of what I mean.